openmc/deplete/abc.py

```diff
@@ -592,6 +592,7 @@ def __init__(self, operator, timesteps, power=None, power_density=None,
                         self._num_stages))
         self.operator = operator
         self.chain = operator.chain
+        self.timestep_units = timestep_units
```

Comment:
This approach won't always work because in some cases the `timestep_units` argument is ignored, e.g.:
```Python
Integrator(timesteps=[(3., 'd'), (1000, 's'), (15, 'h')], ...)
```
What I would suggest is to just have `timestep_units` be an argument on the `plot` method.

Reply:
I have now add ```timestep_units``` as an argument on the plot method.